src/index.ts

```diff
@@ -103,8 +101,25 @@ class ColorPickerWidget extends WidgetType {
     picker.type = 'color';
     picker.value = this.color;
 
+    picker.style.display = 'none';
+
+    const button = document.createElement('button');
+    button.onclick = () => picker.click();
+
+    // can this be fixed?
+    button.style.height = '1em';
+    button.style.width = '1em';
+    button.style.padding = '0px';
+    button.style.border = '1px solid #4e5569';
+    button.style.borderRadius = '2px';
+    button.style.marginLeft = '0.5ch';
```

Comment:
Out of curiosity, why use `ch` here and `em` for height and width

Reply:
In retrospect, I'm not actually sure if it does work properly, but the intention was that it would be sized in accordance with the text it's annotating - so that it is the same height as the text, and the space next to it is relative to the glyph width. A lot of the values there are quite arbitrary though.